forgevtt-module.js

```diff
@@ -1791,40 +1786,65 @@ class ForgeAPI {
 }
 
 class ForgeCompatibility {
+
+    /**
+     * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
+     * @param {string} version The version to check
+     * @param {string} target The version to check against
+     * @returns {boolean} True when the version is newer than the target, false otherwise
+     */
+    static _isNewerVersion(version, target) {
+        try {
+            return foundry.utils.isNewerVersion(version, target);
+        } catch (_) {
+            return isNewerVersion(version, target);
+        }
+    }
+
+    static get isNewerVersion() {
+        return ForgeCompatibility._isNewerVersion
+    }
```

Comment:
Why'd we need two static methods for this?

Reply:
Seems like a holdover from the implementation in `ForgeVTT`, but I agree it doesn't look necessary anymore.